mlcomp/parallelm/mlpiper/main.py

```diff
@@ -143,7 +143,7 @@ def main(bin_dir=None):
     if not options:
         return
 
-    logging.basicConfig(level=options.logging_level)
+    logging.basicConfig(level=options.logging_level, format='%(asctime)-15s %(levelname)s %(name)s:  %(message)s')
```

Comment:
there is an extra space before last message

Reply:
It was written as is intentionally - it makes the logs more readable